src/Controllers/class-packlink-frontend-controller.php

```diff
@@ -50,6 +51,7 @@ class Packlink_Frontend_Controller extends Packlink_Base_Controller {
 	 * @var array
 	 */
 	private static $help_urls = array(
+		'EN' => 'https://support-pro.packlink.com/hc/en-gb/articles/210158585-Install-your-WooCommerce-module-in-5-steps',
```

Comment:
In the future we could consider use https://support-pro.packlink.com/ as global constant

Reply:
We could if we get information that for now on, all links would be from this domain. On the other hand, it is needed only here, so no need for a global constant.